aclgen.py

```diff
@@ -434,7 +434,11 @@ def WriteFiles(write_files):
     output.flush()
 
 
-def main(_):
+def main(args):
```

Comment:
True.  FLAGS is global state, I prefer code to not to have to worry about global state.  We can remove this and do what you suggest if preferred.